torchci/scripts/backfillJobs.mjs

```diff
@@ -17,6 +18,26 @@ function getDynamoClient() {
   );
 }
 
+function getClickhouseClient() {
+  return createClient({
+    host: process.env.CLICKHOUSE_HUD_USER_URL ?? "http://localhost:8123",
```

Comment:
nit: A comment explaining the 8123 or linking to their docs could be helpful here.

Reply:
Its the default value when running locally and not using TLS (?) https://clickhouse.com/docs/en/integrations/javascript#url-configuration

Either way I got rid of it since it doesn't make much sense to run this script without these environment variables set to actual values